platform-tests/src/test/java/org/junit/platform/commons/util/ClassNamePatternFilterUtilsTests.java

```diff
@@ -169,4 +169,155 @@ void alwaysExcludedClassName(String pattern) {
 				.isEmpty();
 	}
 
+	//@formatter:off
+	@ValueSource(strings = {
+			"org.junit.jupiter.*",
+			"org.junit.platform.*.NonExistentClass",
+			"*.NonExistentClass*",
+			"*NonExistentClass*",
+			"AExecutionConditionClass, BExecutionConditionClass"
+	})
```

Comment:
I'm wondering whether we should reduce the duplication in input parameters in this class by defining local composed anotations (`@VanillaClassMatches` etc.), switching to `@MethodSource`, etc. OR if the duplication is ok so one can directly see which patterns are used. WDYT?

Reply:
Well, in my opinion, clarity is crucial when it comes to patterns. Therefore, I believe it would be better to maintain the current approach. 

Alternatively, how about completing the work using the current method, and then working on reducing duplication using approaches like `@MethodSource` afterwards?"